internal/util/logging.go

```diff
@@ -0,0 +1,27 @@
+package util
```

Comment:
Is this used?

Reply:
(meaning the whole file)

Reply:
oh crap - nah that snuck in

Reply:
was also trying to think about logging then gave up - ty!